data-raw/internalData.R

```diff
@@ -15,7 +15,26 @@ atm2paC <- 101300
 km <- 1000
 
 wgs84 <- 4326
+
+# SSHS constants
+
 sshs <- c(18, 33, 42, 49, 58, 70)
+SSHS_PALETTE = c("#00CCFF",
```

Comment:
```suggestion
sshsPalette = c("#00CCFF",
```

Reply:
Ok done, I'm removing the upper case one